src/.buildkite/test/run.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 
 cd src
 
-npm i --silent -g pnpm@5.1.7
+npm i --silent -g pnpm@5.1.7 esbuild@0.7.1 --unsafe-perm
```

Comment:
You're using --unsafe-perm here but not in the publish one?

Reply:
Oops